build-windows/build.sh

```diff
@@ -5,7 +5,7 @@ set -e
 # Config
 
 export SCONS="scons -j${NUM_CORES} verbose=yes warnings=no progress=no"
-export OPTIONS="production=yes angle_libs=/root/angle"
+export OPTIONS="production=yes angle_libs=/root/angle mesa_libs=/root/mesa d3d12=yes"
```

Comment:
Don't we need `d3d12=yes` too? Or are we going to make it the default?

Reply:
> Don't we need d3d12=yes too? 

We do, and it is there (last param). We do not need `dxc_path` to build, it's only used to copy dll (headers and in the `thirdparty`).

Reply:
Oh right I missed it :D